components/HotWallet/plugins/SaberStats/SaberStats.tsx

```diff
@@ -0,0 +1,33 @@
+import useSaberStats from '@hooks/useSaberStats'
+import { SupportIcon } from '@heroicons/react/outline'
+import SaberStat from './SaberStat'
+import { HotWalletAccount } from '@hooks/useHotWallet'
+
+const HotWalletPluginSaberStats = ({
+  hotWalletAccount,
+}: {
+  hotWalletAccount: HotWalletAccount
+}) => {
+  const { saberStats } = useSaberStats(hotWalletAccount)
+
+  if (!saberStats) {
+    return <></>
+  }
+
+  return (
+    <div className="bg-bkg-2">
+      <h3 className="bg-bkg-2 mb-4 flex items-center">
+        <SupportIcon className="flex-shrink-0 h-5 mr-1 text-primary-light w-5" />
+        Saber Stats
+      </h3>
+
+      <div style={{ maxHeight: '350px' }} className="overflow-y-auto space-y-3">
+        {saberStats.map((saberStat) => (
+          <SaberStat key={saberStat.liquidityPoolName} saberStat={saberStat} />
+        ))}
+      </div>
+    </div>
+  )
```

Comment:
```suggestion

  return !saberStats ? null : (
    <div className="bg-bkg-2">
      <h3 className="bg-bkg-2 mb-4 flex items-center">
        <SupportIcon className="flex-shrink-0 h-5 mr-1 text-primary-light w-5" />
        Saber Stats
      </h3>

      <div style={{ maxHeight: '350px' }} className="overflow-y-auto space-y-3">
        {saberStats.map((saberStat) => (
          <SaberStat key={saberStat.liquidityPoolName} saberStat={saberStat} />
        ))}
      </div>
    </div>
  )
```

Reply:
For this precise point, I do prefer the first option of leaving straight away. rather than add an indentation level. But it's really a personal taste.